falcon_mutualauth/mutualauth.py

```diff
@@ -74,11 +74,35 @@ def __init__(self, *default_roles):
     # NOTE(fxfitz): Falcon v1.0 requires the params parameter, so adding this
     # here now to be forward compatible with Falcon when v1.0 is released
     def process_resource(self, req, resp, resource, params=None):
-        authorized_roles = getattr(
-            resource, 'authorized_for', self._default_roles)
+        authd_roles = getattr(resource, 'authorized_for_map', {})
```

Comment:
Since sometimes specific resources aren't going to have role-based access  and will depend on the api-wide default_roles passed to to Authorize